rust/azure_iot_operations_mqtt/src/session.rs

```diff
@@ -658,10 +700,17 @@ impl SessionExitHandle {
     /// has ended.
     ///
     /// Returns true if the exit was graceful, and false if the exit was forced.
-    #[allow(clippy::unused_async)] // TODO: remove once implemented
-    pub async fn exit_force(&self) -> bool {
+    #[allow(clippy::must_use_candidate)]
+    pub fn force_exit(&self) -> bool {
         // TODO: once this is implemented, change METL tests back to using this instead of try_exit().unwrap()
-        unimplemented!()
+
+        if self.try_exit().is_ok() {
+            true
+        } else {
+            log::debug!("Session not connected, forcing exit immediately");
```

Comment:
thoughts, since I'd consider this more of a major state change log than a log that could get spammy?
```suggestion
            log::info!("Session not connected, forcing exit immediately");
```